IntegrationTests/PRMEMC/script/compileHLS.tcl

```diff
@@ -0,0 +1,39 @@
+# Common settings
+source ../../../project/env_hls.tcl
+set CFLAGS {-std=c++11 -I../../../TrackletAlgorithm -I../../../TopFunctions}
+set cwd [pwd]
+
+# Build PR_L3PHIC
+open_project -reset projrouter
+add_files ../../../TopFunctions/ProjectionRouterTop.cc -cflags "$CFLAGS"
+set_top ProjectionRouterTop_L3PHIC
+open_solution solution_L3PHIC
+cd ../../../project
+source settings_hls.tcl
+cd $cwd
+csynth_design
+export_design -format ip_catalog
+
+# Build ME_L3*
+open_project -reset matchengine
+add_files ../../../TopFunctions/MatchEngineTop.cc -cflags "$CFLAGS"
+set_top MatchEngineTop_L3
```

Comment:
If all of PR, ME, and MC used the test bench pattern in #203 then you wouldn't need to hard code L3. Maybe just something to keep in mind for the future.

Reply:
Wouldn't you though? You still have to let HLS know which function to use for C-synthesis.